core-concepts/streams/stream-names.md

```diff
@@ -39,7 +39,7 @@ For example, the stream name for all commands for the `account` category would b
 
 ## Many Category Types
 
-A stream name can have more than one category type.
+A stream name can have more than one category type.  To achieve this you use the `+` symbol to join together multiple category types.
```

Comment:
The "to achieve this you" seems superfluous in communicating the fact in-question. Also, "you use the" addresses the reader of the document rather than the subject. It's something that ideally should be avoided when possible, especially in technical writing.

"The `+` symbol joins separates the category types when there are more than one" ought to suffice. Note: The types aren't joined. The result isn't a concatenation.